Improvements to Lazy Input, Buttons for Inserting Rows

mob1lity opened the pull request:
– Dot as a separator: possibility to separate time values with "." instead of ":" 
– No separator mode: possibility to quickly input time values without any separators at all 
– Option to automatically input data through filling out any or both of form fields 
– Option to easily add Resets: write or select a level, tick the checkbox "Reset?" and click the "Add" button 
– All comment fields now have predefined Hitman Levels as options to select. And are also formatted to prevent some link issues.

mob1lity committed:
Some improvements.
– Dot as a separator: possibility to separate time values with "." instead of ":"
– No separator mode: possibility to quickly input time values without any separators at all
– Option to automatically input data through filling out any or both of form fields
– Option to easily add Resets: write or select a level, tick the checkbox "Reset?" and click the "Add" button
– All comment fields now have predefined Hitman Levels as options to select. And are also formatted to prevent some link issues.

solderq35 committed:
try to add x buttons to fork

solderq35 commented:
Confirmed that mob1lity's changes work as intended, and also added "X" buttons per row (since by default you cannot click on the dropdown menu per row until the row is empty).

Also noted that the "add reset split button" will just find what is the first empty row (both time and split label left blank), nearest to the top.

![image](https://user-images.githubusercontent.com/82061589/233993604-2537f16e-9e59-4b42-ad9d-2de773464852.png)

solderq35 commented:
Closes https://github.com/solderq35/fg-time-calc/issues/3

mob1lity commented:
> Confirmed that mob1lity's changes work as intended, and also added "X" buttons per row (since by default you cannot click on the dropdown menu per row until the row is empty).
> 
> Also noted that the "add reset split button" will just find what is the first empty row (both time and split label left blank), nearest to the top.
> 
> ![image](https://user-images.githubusercontent.com/82061589/233993604-2537f16e-9e59-4b42-ad9d-2de773464852.png)

Ye, so basically if you set both Time and Comment then it will add it to a nearest empty row. If you set only one of those fields then it will look up for first empty cell of that type. For example if you use presets for Seasons/Trilogy you can just set value for Time field then click Add and eventually fill out table from top to bottom.
